cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/handler/async/AddBrokerRequest.java

```diff
@@ -5,16 +5,17 @@
 package com.linkedin.kafka.cruisecontrol.servlet.handler.async;
 
 import com.linkedin.kafka.cruisecontrol.async.OperationFuture;
-import com.linkedin.kafka.cruisecontrol.servlet.KafkaCruiseControlServlet;
 import com.linkedin.kafka.cruisecontrol.servlet.parameters.AddBrokerParameters;
+import java.util.Map;
+
+import static com.linkedin.kafka.cruisecontrol.servlet.parameters.ParameterUtils.ADD_BROKER_PARAMETER_OBJECT_CONFIG;
 
 
 public class AddBrokerRequest extends AbstractAsyncRequest {
-  private final AddBrokerParameters _parameters;
+  private AddBrokerParameters _parameters;
 
-  public AddBrokerRequest(KafkaCruiseControlServlet servlet, AddBrokerParameters parameters) {
-    super(servlet);
-    _parameters = parameters;
+  public AddBrokerRequest() {
```

Comment:
maybe we do not need to explicitly specify default constructor here(apply to other `Request` class)?

Reply:
To create an instance of such subclasses using `KafkaCruiseControlConfig#getConfiguredInstance(String, Class<T>, Map<String, Object>`, or any other methods that call `Utils#newInstance(Class<T>)`, the class needs to explicitly specify a public no-argument constructor.